internals/api/secret_version_test.go

```diff
@@ -44,8 +44,9 @@ func TestCreateSecretVersionRequest_Validate_MaxSize(t *testing.T) {
 			t.Fatal(err)
 		}
 
+		id := uuid.New()
 		r := CreateSecretVersionRequest{
-			SecretKeyID:   uuid.New(),
+			SecretKeyID:   &id,
```

Comment:
Same here, is the `SecretKeyID` field optional and should it therefore be a pointer?